server/chat-plugins/randombattles/index.ts

```diff
@@ -850,15 +857,14 @@ export const commands: Chat.ChatCommands = {
 		}
 
 		if (!movesets.length) {
-			if (isInexact) {
-				return this.errorReply(`Error: No Pok\u00e9mon named '${args[0].trim()}' was found. (Check your spelling?)`);
-			}
+			this.sendReply(inexactMsg)
 			return this.errorReply(`Error: ${species.name} has no Random Battle data in ${format.name}`);
 		}
 		let buf = movesets.join('<hr/>');
 		if (setCount <= 2) {
 			buf = buf.replace(/<details>/g, '<details open>');
 		}
+		this.sendReply(inexactMsg);
```

Comment:
To keep it most consistent with the ``/dt`` command, the message appears outside the ``replyBox`` area.  This means that there are two ``sendReply`` calls.  The alternative is to remove the call to ``errorReply`` and ``sendReplyBox`` and construct the reply string manually to send a singular reply.

Since I wasn't aware of any issues with just using ``sendReply``/``sendReplyBox``/``errorReply`` twice, I opted for this form.